src/libraries/System.Threading.Thread/ref/System.Threading.Thread.cs

```diff
@@ -77,7 +77,7 @@ public sealed partial class Thread : System.Runtime.ConstrainedExecution.Critica
         public bool Join(int millisecondsTimeout) { throw null; }
         public bool Join(System.TimeSpan timeout) { throw null; }
         public static void MemoryBarrier() { }
-        [System.ObsoleteAttribute("Thread.ResetAbort is not supported and throws PlatformNotSupportedException.", DiagnosticId = "SYSLIB0006", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
+        [System.ObsoleteAttribute("Thread.Abort is not supported and throws PlatformNotSupportedException.", DiagnosticId = "SYSLIB0006", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
This seems wrong - This method is `ResetAbort`.

Reply:
Right but the source had Thread.Abort as the message probably to avoid an additional obsoletion message: https://github.com/dotnet/runtime/blob/95f7f7a026d74e0720d0dfdf2de799933b832df2/src/libraries/System.Private.CoreLib/src/System/Threading/Thread.cs#L423.

Reply:
I think that was an oversight in the PR. It makes more sense for the message to refer to the current method, not some other method.

cc @kouvel @GrabYourPitchforks

Reply:
Changed the message in both the ref and the src but the aka.ms link still refers to Thread.Abort: https://docs.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0006.